src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -3186,7 +3281,10 @@ public static int getCacheLoadTimeout()
     @VisibleForTesting
     public static void setCacheLoadTimeout(int seconds)
     {
-        conf.cache_load_timeout = new DurationSpec.IntSecondsBound(seconds);
+        if (seconds < 0)
```

Comment:
minor nit: 
```suggestion
        if (seconds <= 0)
```

Reply:
See also https://github.com/apache/cassandra/pull/1754#discussion_r931567701

Reply:
Yeah, makes sense not to do conversions, absolutely, thanks. I will also add more tests as @maedhorz suggested, I am actually working on special cases loading from yaml and from fromMap (old and new names) and checking the Settings Virtual Table for all Converters. And I think this is exactly what you are looking for here. I will share commit later today